{"uid":"668a7e26dfcc38de","name":"test_setup","fullName":"tests.integration.test_smoke#test_setup","historyId":"d623373ae070a62cab8e4b2f9dd522ae","time":{"start":1757035451991,"stop":1757035659291,"duration":207300},"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: sysbench/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 33, in test_setup\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3245, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3136, in _raise_for_status\n    raise error_type(\njuju.errors.JujuUnitError: Unit in error: sysbench/0","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1757035451703,"stop":1757035451703,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1757035451703,"stop":1757035451704,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1757035451704,"stop":1757035451990,"duration":286},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1757035451990,"stop":1757035451990,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: sysbench/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 344, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 246, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 900, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 263, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 178, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1671, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 157, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 33, in test_setup\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3245, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3136, in _raise_for_status\n    raise error_type(\njuju.errors.JujuUnitError: Unit in error: sysbench/0","steps":[],"attachments":[{"uid":"4e0548e653eb174f","name":"log","source":"4e0548e653eb174f.txt","type":"text/plain","size":20425}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1757035659309,"stop":1757035659309,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1757035659310,"stop":1757035660020,"duration":710},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1757035660023,"stop":1757035660023,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1757035660023,"stop":1757035660023,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1757035660023,"stop":1757035660024,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"pkrvm7jw40e0xgp"},{"name":"thread","value":"16454-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"dba4c20dc104e8f5","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":71,"skipped":0,"passed":353,"unknown":9,"total":433},"items":[{"uid":"cab92dc28876870a","reportUrl":"../2262//#testresult/cab92dc28876870a","status":"passed","time":{"start":1756949071195,"stop":1756949684304,"duration":613109}},{"uid":"c67ba8746964c576","reportUrl":"../2260//#testresult/c67ba8746964c576","status":"passed","time":{"start":1756862723076,"stop":1756863330605,"duration":607529}},{"uid":"626a2bac5ebb20ed","reportUrl":"../2254//#testresult/626a2bac5ebb20ed","status":"passed","time":{"start":1756776271036,"stop":1756776833277,"duration":562241}},{"uid":"6aab1c520f306617","reportUrl":"../2253//#testresult/6aab1c520f306617","status":"passed","time":{"start":1756689921169,"stop":1756690497313,"duration":576144}},{"uid":"6336be5004da335c","reportUrl":"../2252//#testresult/6336be5004da335c","status":"passed","time":{"start":1756603478687,"stop":1756604087850,"duration":609163}},{"uid":"c6d866c646c9d643","reportUrl":"../2251//#testresult/c6d866c646c9d643","status":"passed","time":{"start":1756517035532,"stop":1756517608696,"duration":573164}},{"uid":"5f690d0d532a45c8","reportUrl":"../2249//#testresult/5f690d0d532a45c8","status":"passed","time":{"start":1756430713652,"stop":1756431260942,"duration":547290}},{"uid":"700b99ba87fe1ff2","reportUrl":"../2247//#testresult/700b99ba87fe1ff2","status":"passed","time":{"start":1756344385158,"stop":1756344952952,"duration":567794}},{"uid":"554551a39391aadb","reportUrl":"../2245//#testresult/554551a39391aadb","status":"passed","time":{"start":1756257900485,"stop":1756258455683,"duration":555198}},{"uid":"bafb6cdaa1e3ae17","reportUrl":"../2240//#testresult/bafb6cdaa1e3ae17","status":"passed","time":{"start":1756171509671,"stop":1756172078138,"duration":568467}},{"uid":"ac62761a75aaad2c","reportUrl":"../2238//#testresult/ac62761a75aaad2c","status":"passed","time":{"start":1756085079647,"stop":1756085709589,"duration":629942}},{"uid":"b58ef216d3cc29a6","reportUrl":"../2236//#testresult/b58ef216d3cc29a6","status":"passed","time":{"start":1755998663705,"stop":1755999374921,"duration":711216}},{"uid":"d2b768153544dea9","reportUrl":"../2235//#testresult/d2b768153544dea9","status":"passed","time":{"start":1755912317563,"stop":1755912935035,"duration":617472}},{"uid":"12f38ea7d82fab3b","reportUrl":"../2233//#testresult/12f38ea7d82fab3b","status":"passed","time":{"start":1755825832577,"stop":1755826493057,"duration":660480}},{"uid":"dfcd65772aba379a","reportUrl":"../2232//#testresult/dfcd65772aba379a","status":"passed","time":{"start":1755739649880,"stop":1755740208584,"duration":558704}},{"uid":"58f7e56f410432c1","reportUrl":"../2230//#testresult/58f7e56f410432c1","status":"passed","time":{"start":1755653072029,"stop":1755653649187,"duration":577158}},{"uid":"36525913c7067e09","reportUrl":"../2226//#testresult/36525913c7067e09","status":"passed","time":{"start":1755566667833,"stop":1755567323415,"duration":655582}},{"uid":"770a800fbec6616b","reportUrl":"../2224//#testresult/770a800fbec6616b","status":"passed","time":{"start":1755480354749,"stop":1755480954150,"duration":599401}},{"uid":"e0dcdfc3a061afa3","reportUrl":"../2222//#testresult/e0dcdfc3a061afa3","status":"passed","time":{"start":1755393856232,"stop":1755394758892,"duration":902660}},{"uid":"499a1e03b6710311","reportUrl":"../2221//#testresult/499a1e03b6710311","status":"passed","time":{"start":1755307434650,"stop":1755308082565,"duration":647915}}]},"tags":["asyncio","abort_on_fail"]},"source":"668a7e26dfcc38de.json","parameterValues":[]}